crates/swift-bridge-ir/src/errors/parse_error.rs

```diff
@@ -5,6 +5,7 @@ use syn::{ForeignItemFn, ForeignItemType, LitStr};
 use syn::{Token, Type};
 
 // <!-- ANCHOR: mdbook-parse-error-enum -->
+#[allow(dead_code)] // TODO: remove
```

Comment:
Let's remove this

Reply:
I added it there because I didn't think it was a good idea to remove the enum attribute error, and currently since error handling isn't implemented it's unused and therefore generates a warning.